src/applications/vaos/components/FacilityHours.jsx

```diff
@@ -1,15 +1,28 @@
-// Dependencies
 import React from 'react';
-// Relative
-import { formatOperatingHours } from '../utils/formatters';
+import moment from 'moment';
 
-/**
- * VA Facility Known Operational Hours
- */
-const FacilityHours = ({ location }) => {
-  // Derive the formatted hours info.
-  const hoursInfo = location?.hours;
+function getHourText(hours) {
+  if (!hours) {
+    return 'Closed';
+  }
 
+  if (hours.allDay) {
+    return '24/7';
+  }
+
+  if (!hours.openingTime && !hours.closingTime) {
+    return '';
```

Comment:
If there's no opening and closing time and the location isn't open all day, should we return 'Closed'?

Reply:
When both of these are blank, it means we didn't see the text that normally means closed and we also didn't have parseable dates. I was trying to follow the existing logic as much as possible, which is that closed is either a string with "close" or "-" in it.